department-of-veterans-affairs/va.gov-cms
# VACMS-2975 (2): Re-save health services nodes to update titles/DB aliases

## Description

NB: This should not be merged until #3087  has been merged and is live.

See #2975.

## Testing done

Tested DB post update hook locally using new prod DB dump.

## Screenshots


## QA steps

Verify that Facility/System Health Service node paths are correct after the database update has been completed.

- Go to the content listing, filter by Content Type: VAMC Facility Health Service, visit a few representative pages:
  - [ ] Validate that paths match the pattern "{Parent Facility node URL}/{Regional health service name & description}"
- Go to the content listing, filter by Content Type: VAMC System Health Service, visit a few representative pages:
  - [ ] Validate that paths match the pattern "{Parent Region node URL}/health-services/{node title}"
- [ ] validate that the hook_update_n completed 
![image](https://user-images.githubusercontent.com/5752113/94292422-f0b00c00-ff2a-11ea-8fab-c164723ed0cd.png)
- [ ] validate that  ~1400 nodes were updated via the hook_update_n
- [ ] Validate that updated nodes have proper revision message 
![image](https://user-images.githubusercontent.com/5752113/94292955-a5e2c400-ff2b-11ea-860c-468804143e4b.png)


## Definition of Done
